pkg/node/node.go

```diff
@@ -753,14 +777,14 @@ func NewBee(addr string, publicKey *ecdsa.PublicKey, signer crypto.Signer, netwo
 	var chainSyncer *chainsyncer.ChainSyncer
 
 	if o.FullNodeMode {
-		cs, err := chainsync.New(p2ps, swapBackend)
+		cs, err := chainsync.New(p2ps, swapChain)
```

Comment:
btw this whole `swap` prefix to the chain related stuff is outdated and can be generalized. the swapChain is the chain backend that serves all components, not just SWAP

Reply:
got rid of it